beacon-chain/node/node.go

```diff
@@ -404,9 +405,19 @@ func (b *BeaconNode) registerSyncService(ctx *cli.Context) error {
 		return err
 	}
 
-	var initSync *initialsync.Service
-	if err := b.services.FetchService(&initSync); err != nil {
-		return err
+	var initSync prysmsync.Checker
+	if cfg := featureconfig.Get(); cfg.EnableInitSyncQueue {
+		var initSyncTmp *initialsync.Service
+		if err := b.services.FetchService(&initSyncTmp); err != nil {
+			return err
+		}
+		initSync = initSyncTmp
+	} else {
+		var initSyncTmp *initialsyncv1.Service
```

Comment:
isn't v1 suppose to be newer? wouldnt that be the one that's gated?
perhaps a better name?

Reply:
I've assumed that I was working on v2 :) considered putting the old version under v0 tag, but supposed we have very solid working version not some prototype, so went with v1. Please let me know if you think that we need to change it.

Reply:
yeah I see the confusion since we don't put v2 for the new service, perhaps: 

s/initialsyncv1/initialsyncOld. ?

Reply:
Sounds good, will do it now.

Reply:
Done!